js/run.js

```diff
@@ -174,6 +174,24 @@ function parseOutput(res, o, oTitle)
         o.text(output);
 }
 
+// wraps a unittest into a runnable script
+function wrapIntoMain(code) {
```

Comment:
A bit annoying that this logic is duplicated in the JS code and in `dspec_tester.d`, but I guess that's not easily avoidable her.